node/tests/SharedTests.ts

```diff
@@ -2775,6 +2775,68 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `expiretime and pexpiretime test_%p`,
+        async (protocol) => {
+            await runTest(async (client: BaseClient, cluster) => {
+                if (cluster.checkIfServerVersionLessThan("7.0.0")) {
+                    return;
+                }
+
+                const key = uuidv4();
+
+                expect(await client.set(key, "foo")).toEqual("OK");
+                expect(await client.ttl(key)).toEqual(-1);
+
+                if (cluster.checkIfServerVersionLessThan("7.0.0")) {
```

Comment:
Check here is redundant because we are checking for version in the beginning of the test.

Reply:
i agree but it will align to both the python and java tests.